TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    filtered_data_products = Observations.filter_products(data_products,productSubGroupDescription="LC",productType = "SCIENCE",productGroupDescription = "Minimum Recommended Products")
</pre><p> </p><p>Let's add more narrative about our selection here. What is an "LC" file? (a pre-made light curve processed by the TESS pipeline. see previous Notebook for more info about this process)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>added a markdown cell on this before this cell</p>